scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -284,7 +286,7 @@ library_ids <- c(
   "SCPCL000491"
 )
 
-# subset sce to only contain samples with IDs of interest 
+# subset sce to only contain samples of interest 
 samples_to_keep <- integrated_sce$sample %in% library_ids
 rms_sce <- integrated_sce[,samples_to_keep]
 
```

Comment:
For the next `remove sce` chunk, I'd take the comment out of the chunk and turn into regular text as bit clearer transition to the removal.